Scientific Events Ontology (SEO)

saidfathalla opened the pull request:
Dear all,
This is the Scientific Events Ontology (SEO), a reference ontology for modelling data about scientific events such as conferences, symposioums and workshops

dgarijo commented:
@saidfathalla 
w3id repo is meant to host the redirections, not the actual OWL files you are redirecting to. Please remove those files and point to them in your server or repository

dgarijo commented:
Also, you seem to have copied pasted the htaccess from another repo, and you are creating w3ids such as w3id.org/SemSur/browserquest

davidlehn commented:
Also remove the html.  Are those SemSur files supposed to be in this request?  If they are, the redirects are setup for `https://w3id.org/SemSur/semsur`, which I'm guessing is wrong?

Ideally rebase or squash this to remove the extra commits, but we can squash things if needed when merging.

saidfathalla commented:
@dgarijo @davidlehn 
I hope everything is fine now.

davidlehn commented:
- This is changing the `un/.htaccess` file.
-   Also just to check, since it wasn't implied by the title, you are adding the SemSur directory on purpose here?
- Did you test these files?
- These rules will result in https://w3id.org/seo/seo and https://w3id.org/SemSur/semsur.  Is that what you want?

saidfathalla commented:
I do not know how this change un/.htaccess file? thats why i revert deleting semsur.
What I need is only redirect to seo by the URL: https://w3id.org/seo .

saidfathalla commented:
@davidlehn @dgarijo I hope everything is fine now. Please pay attention to this request due to my time limitation. Thanks

davidlehn commented:
You are still changing `un/.htaccess`.  Please revert that change.
https://github.com/perma-id/w3id.org/pull/1308/files

Also confirm you really want those double `seo/seo` type paths.  In a directory called "foo" most people want to use a `^$` pattern for the current dir vs a sub-path `^foo$`.

saidfathalla commented:
@davidlehn @dgarijo , I have changed the.htacces files of semsur and seo to redirected from https://w3id.org/semsur and https://w3id.org/seo  respectively. I also reverted the access to un/.htaccess.
is it OK now?

davidlehn commented on SemSur/.htaccess:
```diff
@@ -0,0 +1,27 @@
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ http://sda.tech/SemSur/Documentation/Semsur.html  [R=303,NE,L]
+RewriteCond %{HTTP_ACCEPT} ^.applicationrdf+xml. 
```
That pattern looks wrong.  Perhaps this would work:
```suggestion
RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.*
```

saidfathalla commented:
@davidlehn Thanks for the suggestion. Is it OK now?

saidfathalla commented:
@davidlehn @dgarijo , could you please check?
